features/unattached_commands.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Command behaviour when unattached
+
+    Scenario: Unattached detach in a trusty lxd container
+        Given a trusty lxd container
+          And ubuntu-advantage-client is installed
```

Comment:
are we going to have any tests where we don't have it installed? Or should that be combined with the first? I can see being more verbose is helpful, but it seems this would be a lot more cut and paste.

If we want to have ua client not installed then maybe have one that removes it?

Reply:
We'll also want this to be a single step so that we can reuse an image with the right version inside to speed things up (i.e. #854).

Can we defer addressing this until we're also making that change?

Reply:
agreed

Reply:
Thanks!